src/common/options.cc

```diff
@@ -850,88 +872,95 @@ std::vector<Option> get_global_options() {
 
     Option("heartbeat_interval", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(5)
-    .set_description(""),
+    .set_description("Frequency of internal heartbeat checks (seconds)"),
 
     Option("heartbeat_file", Option::TYPE_STR, Option::LEVEL_ADVANCED)
-    .set_default("")
-    .set_description(""),
+    .set_default("File to touch on successful internal heartbeat")
```

Comment:
this should be set_description()?  a file named `'File to touch on successful internal heartbeat'` keeps getting recreated in my build directory :D fixed in https://github.com/ceph/ceph/pull/25617